### DISCUSSION TITLE
Implementing Interactive donut charts and Pie charts

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#1984] [Q&A]
Hi!I want to implement interactive pie and donut charts in Wave. By Interactive, I want the user to be able to click and highlight different sections of the donut, On selecting one section of the donut I want it to be exploded out, and some tabular data to be displayed based on this selection.
I thought of using ChartJs for making these charts. Can I use it in my Wave application? Will the events be captured in my serve function?

**Comment by mturoci** [comment]
Yes, you can use any JS lib you like and register events as [documented](https://wave.h2o.ai/docs/javascript/).
